components/SalesHistoryTable/index.tsx

```diff
@@ -76,14 +123,19 @@ const SalesHistoryTable = ({ tableData, error }: Props): JSX.Element => {
 
   useEffect(() => {
     (async () => {
-      if (tableData.length) {
-        const chainAccounts = tableData.map(({ buyer }) => buyer);
+      if (renderedData.length) {
```

Comment:
Needs another rebase. Sales history table is stuck in the loading state - we should call `setIsLoading(false)` in this `useEffect` to resolve that.

Let's also move this if statement inside of the `try` block.